packages/components/src/components/hds/accordion/item/index.ts

```diff
@@ -45,6 +58,62 @@ export default class HdsAccordionItemComponent extends Component<HdsAccordionIte
     return this.args.containsInteractive ?? false;
   }
 
+  /**
+   * @param toggleTextSize
+   * @type {HdsTextSizes}
+   * @default false
+   */
+  get toggleTextSize() {
+    const size = this.args.size ?? 'large';
```

Comment:
should this be `medium`? also, should this come from `DEFAULT_SIZE`?

Reply:
Indeed; updated to use `DEFAULT_SIZE`